main.py

```diff
@@ -0,0 +1,39 @@
+from colorama import Fore
+
+from readings import Readings
+from data import monthly
+from data import yearly
+
+
+class main:
```

Comment:
Create `main` function. and wrap it under `if` condition to execute the actual code.

```
def main():
    #Your code


if __name__ == '__main__':
    main()
```